### PR TITLE
Tweaks to autolathe design costs

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Tweaks multiple autolathe designs costs, mainly tools and flamethrower.
Full list included here: https://docs.google.com/spreadsheets/d/1wwWD1Ca927MANKbilkPInMrTsRpSrNFD9T2OTQYk1Xw/edit?usp=sharing
Cost tweaks suggestions of any items are welcome.
## Why It's Good For The Game

Multiple autolathe items are extremely cheap even without upgrades (full upgrade is 12,5% of the original cost).
Giving them non laughable costs and balanced with everything else we have is good.

## Testing

- Did print and verify some items costs, all looks good.
## Changelog
:cl:
tweak: Multiple autolathe design costs have been tweaked
/:cl:

### CODE REVIEW

**Comment by Xsingular** [comment]
Disclaimer: I'm just a casual viewer, and I haven't looked into most other items that closely, but one thing did stand out for me:

The Glass being reduced to 100 Glass makes it a far superiour choice to Small Beakers.
Both containers have the same capacity of 50u, and they both used to cost the same amount of glass.

With this tweak, your are paying five times as much glass for an identical container, the one difference being that Beakers have all the interactable actions of /reagent_container, so you can put a lid on it, or spalsh it on someone. Still a pretty high price difference to pay for those two functions (that might be added to glasses eventually anyway)

Maybe both Small Beakers and Beakers need realignment as well with new costs, or perhaps their inflated costs were there specifically because they can be used to make grenades (something the much more accessible and larger buckets cannot be used for), I'll leave this anomaly to the design and balancing team to look at.

**Comment by Henri215** [comment]
> 

Disclaimer: casual viewer or not, comments and suggestions and sacred

Its a good point that i havent thought about it yet, but in practice i dont think it will make much of a difference since both items are really cheap (even more with upgrades).

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -498,7 +498,7 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe/cable_restrain
 	w_class = WEIGHT_CLASS_SMALL
 	throw_speed = 2
 	throw_range = 5
-	materials = list(MAT_METAL=10, MAT_GLASS=5)
+	materials = list(MAT_METAL=15, MAT_GLASS=10)
```
Comment (Edan52) [inline code/modules/power/cable.dm:501]:
```suggestion
	materials = list(MAT_METAL = 15, MAT_GLASS = 10)
```
Here and in the other places you've modified it

Comment (Henri215) [reply code/modules/power/cable.dm:501]: Will do

**Review by S34NW — APPROVED** [review]
Code's fine

**Review by farie82 — APPROVED** [review]

**Pull request merged**